fpmsyncd/routesync.cpp

```diff
@@ -698,10 +706,17 @@ void RouteSync::onRouteMsg(int nlmsg_type, struct nl_object *obj, char *vrf)
     }
 
     /* Get nexthop lists */
-    string nexthops = getNextHopGw(route_obj);
-    string ifnames = getNextHopIf(route_obj);
+    string gw_list;
+    string intf_list;
+    string mpls_list;
+
+    if (!getNextHopList(route_obj, gw_list, mpls_list, intf_list))
```

Comment:
`getNextHopList`  always return true do we need this if condition.

Reply:
Done.  function return is changed to void.